game-app/game-core/src/main/java/games/strategy/triplea/delegate/remote/IEditDelegate.java

```diff
@@ -32,6 +32,9 @@
   @RemoteActionCode(3)
   String changePUs(GamePlayer player, int pus);
```

Comment:
What's the consideration to keeping the changePU method around?
I'd be in favor to remove it in favor of the new updated and more generic method.

Reply:
It was left so that we could be compatible with pre-2.6 clients. If that's not a goal, happy to remove.